src/main.rs

```diff
@@ -1,18 +1,42 @@
-use std::env;
 use tokio;
 
 mod nubank;
 
+mod simple_user_input {
+    use std::io;
+    pub fn get_input(prompt: &str) -> String {
+        println!("{}", prompt);
+        let mut input = String::new();
+        match io::stdin().read_line(&mut input) {
+            Ok(_goes_into_input_above) => {}
+            Err(_no_updates_is_fine) => {}
+        }
```

Comment:
Why are you matching over the result if you are not using it? It could be 
```rust
io::stdin().read_line(&mut input);
```

Reply:
It throw a warning:
```
unused `Result` that must be used
`#[warn(unused_must_use)]` on by default
this `Result` may be an `Err` variant, which should be handled
```

Reply:
Should I ignore it or leave the code as it was?

Reply:
No modification needed